### PR TITLE
add angular-ui-tinymce@0.0.18 w/ git auto-update

### LINKED ISSUE #10262: [Request] Add ui-tinymce
**Library name:** ui-tinymce
**Git repository url:** https://github.com/angular-ui/ui-tinymce
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by lcd78706**
close #10262, cc @wesleycho

Pull request for issue: #10262 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/angular-ui/ui-tinymce
 * Official website (optional, not the repository): https://angular-ui.github.io/
 * NPM package url (optional): https://www.npmjs.com/package/angular-ui-tinymce
 * License and its reference: MIT  https://github.com/angular-ui/ui-tinymce/blob/master/LICENSE#L1
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 56
   - Count of stars: 440
   - Count of forks: 332
 * NPM download stats (optional):
   - Downloads in the last day: 273
   - Downloads in the last week: 1303
   - Downloads in the last month: 5096

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by lcd78706** [comment]
@x09326 Please review this PR. Thanks!

File: ajax/libs/angular-ui-tinymce/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "angular-ui-tinymce",
+  "filename": "tinymce.min.js",
+  "version": "0.0.18",
+  "description": "This directive allows you to add a tinymce to your form elements.",
+  "author": "https://github.com/angular-ui/ui-tinymce/graphs/contributors",
```
Comment (x09326) [inline ajax/libs/angular-ui-tinymce/package.json]: @PeterDaveHello 
Is the author field okay with that?

Comment (PeterDaveHello) [inline ajax/libs/angular-ui-tinymce/package.json]: fine

**Comment by x09326** [comment]
@lcd78706 
Please rebase with the latest master branch, thanks.

**Comment by lcd78706** [comment]
@x09326 
I have rebased it with the latest master branch. Please review it again. Thanks!

**Comment by x09326** [comment]
LGTM
@cdnjs/intern2 please help me review it.

**Comment by maruilian11** [comment]
LGTM

File: ajax/libs/angular-ui-tinymce/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "url": "git://github.com/angular-ui/ui-tinymce.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/angular-ui/ui-tinymce.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "tinymce*"
```
Comment (PeterDaveHello) [inline ajax/libs/angular-ui-tinymce/package.json]: What's the reason why to not use `**/*` here?

Comment (lcd78706) [inline ajax/libs/angular-ui-tinymce/package.json]: @PeterDaveHello 
I carelessly wrote the files in "dist" identical to those in "src", and I have modified it.
Please review it again. Thanks!

Comment (PeterDaveHello) [inline ajax/libs/angular-ui-tinymce/package.json]: Next time, please call you mentor first.

**Pull request merged**